shared/constants/chat2/convostate.tsx

```diff
@@ -1088,21 +1088,14 @@ const createSlice: Z.ImmerStateCreator<ConvoState> = (set, get) => {
         })
         const message = ordinal ? messageMap.get(ordinal) : undefined
 
-        let readMsgID: number | undefined
-        if (isMetaGood()) {
```

Comment:
don't bail on bad meta, there are many reasons why this can happen. instead just opt into bailing if we do have it, else just write it